transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -276,11 +276,11 @@ private void invokeExceptionCaught(final Throwable cause) {
         if (isAdded()) {
             try {
                 handler().exceptionCaught(this, cause);
-            } catch (Throwable t) {
+            } catch (Throwable error) {
                 if (logger.isWarnEnabled()) {
                     logger.warn(
-                            "An exception was thrown by a user handler's " +
-                                    "exceptionCaught() method while handling the following exception:", cause);
+                        "An exception '{}' was thrown by a user handler's exceptionCaught() " +
```

Comment:
Not sure logging exception.tostring here is going to help much,   the broken exceptionCaught implementation will still need to be fixed without knowing the line number.